resources/examples/manager/BasicAssetLibrary/tests/fixtures.py

```diff
@@ -97,9 +97,9 @@
             "a_reference_to_a_missing_entity": "bal:///missing_entity",
             "the_error_string_for_a_reference_to_a_missing_entity": (
                 "Entity 'bal:///missing_entity' not found"),
-            "a_malformed_reference_to_a_readable_entity": f"bal:///{an_existing_entity_name}?arg",
-            "the_error_string_for_a_malformed_reference_to_a_readable_entity": (
-                f"BAL does not support query parameters")
+            "a_malformed_entity_reference": f"bal:///",
```

Comment:
[fix] We discussed offline about removing the assumption that BAL doesn't support query parameters, which was added just to exercise the apiComplianceSuite, and instead make use of the pre-existing check for an empty entity path.

This commit does that.

Reply:
That looks spot on I think! Thanks